utils/util_mesh.py

```diff
@@ -357,3 +357,18 @@ def split_edge_at_point_from_closest_vert(edge, vert, split_point):
     split_length = (close_vert.co - split_point).length
     split_factor = split_length / edge.calc_length()
     return bmesh.utils.edge_split(edge, close_vert, split_factor)
+
+def get_width_and_height(face):
```

Comment:
Function `calc_face_dimensions` in `util_mesh.py` does this already. Use it to avoid duplicate code.